mscore/plugin/api/qmlpluginapi.cpp

```diff
@@ -125,7 +126,7 @@ bool PluginAPI::writeScore(Score* s, const QString& name, const QString& ext)
 
 Score* PluginAPI::readScore(const QString& name, bool noninteractive)
       {
-      Ms::Score* score = msc()->openScore(name, true);
+      Ms::Score* score = msc()->openScore(name, (noninteractive ? false : true));
```

Comment:
Why not just `openScore(name, !noninteractive)`?

Reply:
Well.., actually no reason, I guess. ;-) I'll change it.